Vagrantfile

```diff
@@ -43,12 +43,8 @@ Vagrant.configure("2") do |config|
 
   # Allow Mac OS X docker client to connect to Docker without TLS auth
   # https://github.com/deis/deis/issues/2230#issuecomment-72701992
-  config.vm.provision "shell" do |s|
-    s.inline = <<-SCRIPT
-      echo 'DOCKER_TLS=no' >> /var/lib/boot2docker/profile
-      sudo /etc/init.d/docker restart
```

Comment:
Please keep the same format of the provisioner and just replace this line with
`sudo /etc/init.d/docker stop >/dev/null 2>&1 || sudo /etc/init.d/docker start`